### ISSUE TITLE
pt训练时，如果max length > 模型的max position embedding会怎么样

### ISSUE THREAD

**Issue by xindaaW** [modelscope/ms-swift#7018]
pt训练时，如果max length > 模型的max position embedding会怎么样，我目前用如下参数训练时，发现不会报错，但没有配置任何scaling参数，请问目前的框架会进行怎样的处理：
Qwen3-8B-Base max position embedding： 32768

`swift pt \
    --model $model_path \
    --model_type qwen3 \
    --train_type full \
    --dataset ${DATA_PATHS[@]} \
    --val_dataset ${VAL_DATA_PATHS[@]} \
    --dataset_shuffle true \
    --shuffle_buffer_size 10000 \
    --streaming true \
    --torch_dtype bfloat16 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --learning_rate 1e-5 \
    --gradient_accumulation_steps 2 \
    --sequence_parallel_size 8 \
    --packing true \
    --eval_steps 5 \
    --save_steps 500 \
    --save_total_limit 2 \
    --logging_steps 1 \
    --deepspeed zero3 \
    --max_length 64000 \
    --max_steps 10000 \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --dataset_num_proc 8 \
    --save_only_model true \`
